### PR TITLE
update info about parameters

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1527

The Manufacturer related parameters are self-explanatory in my opinion.

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Minor copy editing and style checks needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306@tseward

Peer review needed for this PR. Thanks.

File: docset/windows/trustedplatformmodule/Get-Tpm.md [hunk 1]
```diff
@@ -66,12 +69,16 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 ### TpmObject
 This cmdlet returns a **TpmObject** object that contains the following information:
 
-- TpmReady. Whether a TPM complies with Windows Server® 2012 standards.
 - TpmPresent. Whether there is a TPM on the current computer
+- TpmReady. Whether a TPM complies with Windows Server® 2012 standards.
```
Comment (JohanFreelancer9) [inline docset/windows/trustedplatformmodule/Get-Tpm.md]:
```suggestion
- TpmReady. Whether a TPM complies with Windows Server 2012 standards.
```

**Comment by dariomws306** [comment]
@tseward

**Comment by tiburd** [comment]
Hi, @VLG17,

We are closing this PR because merging to Live is restricted to the PubOps team only. To find out more about the process, see the [PubOps Publishing Process page](https://review.docs.microsoft.com/en-us/office-authoring-guide/pubops-publishing-process?branch=master). If you resubmit this to the Master branch, the PubOps team will review, merge when ready, and publish to Live. Thanks.

**Comment by e0i** [comment]
@VLG17 Please re-submit the PR targeting `master` branch. Thanks.

**Pull request closed without merging**